tripperware.go

```diff
@@ -21,7 +31,16 @@ func (t Tripperwares) RoundTrip(req *http.Request) (*http.Response, error) {
 	return t.DecorateRoundTripper(http.DefaultTransport).RoundTrip(req)
 }
 
-// DecorateRoundTripper will decorate a given http.RoundTripper with the given tripperware collection created by TripperwareStack()
+// DecorateClient will decorate a given http.Client with the given tripperware collection
+func (t Tripperwares) DecorateClient(client *http.Client) *http.Client {
+	if client.Transport == nil {
```

Comment:
no need for this `if` 👍 

it is done here: https://github.com/gol4ng/httpware/pull/5/files#L46